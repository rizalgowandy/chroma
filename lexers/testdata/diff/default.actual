1,2d0
< A
< A
4c2
< C
---
> F
5a4
> E
