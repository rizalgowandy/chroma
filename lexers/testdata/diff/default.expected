[
  {"type":"GenericSubheading","value":"1,2d0\n"},
  {"type":"GenericDeleted","value":"\u003c A\n\u003c A\n"},
  {"type":"GenericSubheading","value":"4c2\n"},
  {"type":"GenericDeleted","value":"\u003c C\n"},
  {"type":"GenericStrong","value":"---\n"},
  {"type":"GenericInserted","value":"\u003e F\n"},
  {"type":"GenericSubheading","value":"5a4\n"},
  {"type":"GenericInserted","value":"\u003e E\n"}
]
